src/main/java/uk/gov/companieshouse/api/accounts/model/rest/smallfull/notes/loanstodirectors/Loan.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.companieshouse.api.accounts.model.rest.smallfull.notes.loanstodirectors;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.api.accounts.model.entity.smallfull.notes.loanstodirectors.LoanBreakdownResourceEntity;
+import uk.gov.companieshouse.api.accounts.model.rest.RestObject;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Loan extends RestObject {
+
+    @JsonProperty("director_name")
+    private String directorName;
+
+    @JsonProperty("description")
+    private String description;
+
+    @JsonProperty("breakdown")
+    private LoanBreakdownResourceEntity breakdown;
```

Comment:
We'll need a `rest` counterpart to this object for use in this class

Reply:
Added a rest resource